tensorflow/python/distribute/mirrored_strategy.py

```diff
@@ -230,34 +230,6 @@ class MirroredStrategy(distribute_lib.Strategy):
     1: <tf.Variable ... shape=() dtype=float32, numpy=1.0>
   }
 
-  `experimental_distribute_dataset` can be used to distribute the dataset across
```

Comment:
This is still from a bad rebase, we don't want to delete these docstrings.

Reply:
Re-added the doc string. Does it do?